lib/challenge.js

```diff
@@ -1,10 +1,43 @@
 'use strict'
 
-const boggle = function boggle () {}
+// const tray = [['a', 'b', 'c', 'd'],
+          //     ['e', 'f', 'g', 'h'],
+            //   ['i', 'j', 'k', 'l'],
+          //    ['m', 'n', 'o', 'p']]
+// const listOne = [[0,0], [1, 1], [2, 2], [3, 3]]
+const boggle = function boggle (tray, list) {
+  let answer = []
+  for (let i = 0; i<list.length; i++) {
+    answer.push(tray[((list[i][0])*4)+(list[i][1])])
+  }
+  return answer.join('');
+}
+
+
+
+const bigBoggle = function bigBoggle() {
```

Comment:
Looks like you forgot to include `tray` and `list` as parameters in the function here.